src/ansys/aedt/core/modeler/cad/polylines.py

```diff
@@ -341,14 +341,15 @@ def __init__(
 
             self._is_closed = close_surface
             self._is_covered = cover_surface
-
+            self._solve_inside = False
```

Comment:
Why are those changes required ?

Reply:
to prevent AEDT to fail in creating Polyline in 2D Modeler